front/app/api/idea_images/keys.ts

```diff
@@ -1,6 +1,6 @@
 import { QueryKeys } from 'utils/cl-react-query/types';
 
-const baseKey = { type: 'image', variant: 'idea' };
+const baseKey = { type: 'image' };
```

Comment:
I think we should keep the variant for the rest of the endpoints - just remove it from `item`. Otherwise we can have cache conflicts when we fix the rest of them.